### PR TITLE
Fix bug when dropping a view

### PR DESCRIPTION
**Pull request by TsReaper**
This PR fixes the exception thrown and synchronizes the views in `Environment` with the views in `TableEnvironment` when dropping a view.

### CODE REVIEW

File: src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java [hunk 1]
```diff
@@ -242,6 +273,8 @@ private SqlCommandParser() {
 
 		DROP_VIEW,
 
+		DROP_VIEW_IF_EXISTS,
```
Comment (godfreyhe) [inline src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java:276]: do not add a new command, `DROP_VIEW_IF_EXISTS` is a special case of `DROP_VIEW`

File: src/test/java/com/ververica/flink/table/gateway/operation/DropViewOperationTest.java [hunk 2]
```diff
@@ -31,23 +31,44 @@
...
+
+		DropViewOperation operation3 = new DropViewOperation(context, "MyView1", true);
+		assertEquals(OperationUtil.AFFECTED_ROW_COUNT0, operation3.execute());
+
+		String[] tables3 = context.getExecutionContext().getTableEnvironment().listTables();
+		assertEquals(0, tables3.length);
+	}
+
 	@Test(expected = SqlExecutionException.class)
 	public void testDropViewTest_ViewNotExist() {
```
Comment (godfreyhe) [inline src/test/java/com/ververica/flink/table/gateway/operation/DropViewOperationTest.java:70]: rename `testDropViewTest_ViewNotExist` to testDropView_ViewNotExist

File: src/main/java/com/ververica/flink/table/gateway/operation/DropViewOperation.java [hunk 3]
```diff
@@ -54,10 +58,12 @@ public ResultSet execute() {
 		// all the remaining views are OK, so do the ExecutionContext rebuilding to avoid breaking the view dependency.
 		Environment newEnv = env.clone();
 		if (newEnv.getTables().remove(viewName) != null) {
+			context.getExecutionContext().getTableEnvironment().dropTemporaryView(viewName);
```
Comment (godfreyhe) [inline src/main/java/com/ververica/flink/table/gateway/operation/DropViewOperation.java:61]: can we simply this logic: let only `CatalogManager` hold temporary views which means only the new added line is necessary.

File: src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java [hunk 4]
```diff
@@ -18,6 +18,7 @@
 
 package com.ververica.flink.table.gateway;
 
+import org.apache.calcite.sql.SqlDrop;
```
Comment (godfreyhe) [inline src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java:21]: reorder the imports

File: src/main/java/com/ververica/flink/table/gateway/operation/DropViewOperation.java (hunk 3 above)
Comment (TsReaper) [reply src/main/java/com/ververica/flink/table/gateway/operation/DropViewOperation.java:61]: This deserves a new PR.

**Comment by godfreyhe** [comment]
LGTM

**Comment by godfreyhe** [comment]
merged

**Pull request closed without merging**